oak_remote_attestation_noninteractive/proto/v1/messages.proto

```diff
@@ -24,10 +24,23 @@ option java_package = "oak.session.noninteractive.v1";
 message PublicKeyInfo {
   // The serialized enclave public key.
   // TODO(#3442): Specify key format.
-  bytes public_key = 1;
+  bytes enclave_public_key = 1;
   // The serialized attestation report binding the public key.
   bytes attestation = 2;
-  // TODO(#3442): Add evidence (endorsements and inclusion proofs) for the attested binary.
+  // The serialized AMD-SEV-SNP certificate.
+  bytes tee_certificate = 3;
+  // The serialized binary attestation evidence.
+  bytes binary_attestation = 4;
+  // The serialized server configuration. The details of the configuration is application-specific.
+  // This field is optional and can be left empty.
+  bytes server_config = 5;
+}
+
+message BinaryAttestation {
```

Comment:
How is this related to `binary_attestation` above?

Reply:
Good catch. This is the correct type of `binary_attestation` above. I'll update.